base_layer/p2p/examples/pingpong.rs

```diff
@@ -249,7 +249,7 @@ async fn update_ui(update_sink: CursiveSignal, mut liveness_handle: LivenessHand
     loop {
         ::futures::select! {
             event = event_stream.select_next_some() => {
-                match *event {
+                match &*event {
```

Comment:
I wonder if the reference and dereference operators can be removed, '&*'.

Reply:
Because the broadcast_channel emits `Arc`s the dereference is to get the object contained and the reference is to prevent a move of the inner object. Often you wont have to dereference first, but if I don't the match complains about a type mismatch (`Arc<LivenessEvent>` matching on `LivenessEvent`)